<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBuddy</class>
 <widget class="QDialog" name="AddBuddy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="presenceBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>288</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>activate this checkbox to SBSCRIBE to the buddies presence state</string>
   </property>
   <property name="text">
    <string>Subscribe buddy's presence</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>263</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameEdit">
      <property name="toolTip">
       <string>Full name or nickname of the buddy</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>SIP URI:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="uriEdit">
      <property name="toolTip">
       <string>the SIP URI of the buddy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
